### DISCUSSION TITLE
Unfair-Gamers.com - A community 10 years later

### DISCUSSION THREAD

**Discussion by daniele-salvagni** [daniele-salvagni/dan.salvagni.io#4] [Comments]
**Comments for the blog post:** https://dsalvagni.com/b/unfair-gamers-com-a-community-10-years-later/

**Comment by vezdebest** [comment]
I was vezdebest in this special community and i am proud of being part of this magical forum called Unfair Gamers. I invite all old members that didn't already joined the whatsapp group to hurry up and contact us. We're still here and we're stronger than ever, surely stronger than InForge stinky community.

**Comment by Dr0p3r** [comment]
I was Dr0p3r, 16 in the ranking 🤣
I created cheats and bots for the metin2 private servers, tool for hacking and more.. We found the founder of Unfair-Gamers after years of research.
UG > InForge, remember!

**Comment by tille** [comment]
I was tille, another user of the community
I recall pretty brilliant solutions made for the guys of the community, I was 15 years old so it was kind of hard for me to contribute enough as I wanted to, but I used to read the codes trying to learn as much as I can.
I really would like to be part of the whatsapp group and meet you guys, if you think is possible let me know how to be added into the group.

**Comment by MarcoLagalla** [comment]
I was active on the forum with the username Zyrel, starting as a simple user and then being part of the staff.
I found today this article and it was really a dive into the past; I was only 14 when I found the forum and it gave me the opportunity to passionate with coding and software development in general.

What made us so different was that in those years sharing knowledge was not so frequent and the few that known how to create cheats and mods were very reluctant to teach to others. In UG everything was free and everyone contributed as it was in his possibility.

Thank you for the experience and good luck to all!

**Comment by lucacodastefano** [comment]
I was Luca360 and this forum was the place where I shared best moments of my teenage with a lot of helpful and clever and funny members, I’ve coded lots of simple and stupid programs and cheats for metin2, where my passion for computer science exploded. This community will have forever a space on my heart.

**Comment by mattiamalonni** [comment]
Cheers from IoSnTia 😀
What a journey on UG!
I should put it on my resume, or on my Linkedin profile 😂
It was an amazing pleasure to share and to be shared by thousands of people and learn in the process.
Sometimes nostalgy take places and i wish to go back coding and talking with those amazing people.

**Comment by zen-leaf** [comment]
Hey everyone, GLyTCH here! I stumbled upon this memorial completely by chance while looking for old forum emojis. It feels unreal! UG still has a dedicated spot in my heart, and it was without a shadow of doubt the single reason my life, career and aspirations took the shape they did, for better and for worse! Thinking it was so long ago makes me feel much older that it has any right to.
